STAN.CLIENT/StanMsgHandlerArgs.cs

```diff
@@ -21,7 +21,28 @@ public class StanMsgHandlerArgs : EventArgs
     {
         StanMsg msg = null;
 
-        internal StanMsgHandlerArgs(StanMsg m)
+        /// <summary>
+        ///  Constructor for generating a StanMsgHandlerArgs object.  Used for application unit testing.
+        /// </summary>
+        /// <remarks>
+        /// Objects of this type are normally generated internally by the NATS streaming client.
+        /// This constructor has been provided to facilitate application unit testing.
+        /// </remarks>
+        /// <param name="data">The message payload.</param>
+        /// <param name="redelivered">True if the message may have been redelivered.</param>
+        /// <param name="subject">Subject of the message.</param>
+        /// <param name="timestamp">Message timestamp, nanoseconds since epoch.(1/1/1970)</param>
+        /// <param name="subscription">Subscription of the message.  May be null.</param>

```

Comment:
Same note as before, the cast in `StanMsg.ctor()` necessitates that this is an `AsyncSubscription`.